README.md

```diff
@@ -18,12 +18,12 @@ $ yarn add --dev native-ext-loader
 
 Update rules entry in the Webpack configuration file:
 
-```javascript
+```js
```

Comment:
I believe that `javascript` is [a valid mode](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2143-L2178) for the code snippets. Did you find other information?

Reply:
`javascript` works. `js` is more concise and is the convention